lib/casbin/core_enforcer.rb

```diff
@@ -140,83 +158,134 @@ def save_policy
 
     alias enabled? enabled
 
+    # changes the enforcing state of Casbin,
+    # when Casbin is disabled, all access will be allowed by the Enforce() function.
+    def enable_enforce(enabled = true)
+      self.enabled = enabled
+    end
+
+    # controls whether to save a policy rule automatically to the adapter when it is added or removed.
+    def enable_auto_save(auto_save)
+      self.auto_save = auto_save
+    end
+
+    # controls whether to rebuild the role inheritance relations when a role is added or deleted.
+    def enable_auto_build_role_links(auto_build_role_links)
+      self.auto_build_role_links = auto_build_role_links
+    end
+
```

Comment:
Maybe we can just use attributes (`enabled=`, `auto_save=` and `auto_build_role_links=`)?
Or this is troublesome?

Reply:
this is used outside of the gem, I think we'll leave it that way for now